stack.yaml

```diff
@@ -1,13 +1,14 @@
 resolver: lts-22.6
-system-ghc: true
+system-ghc: false
 nix:
   packages:
-    - secp256k1
```

Comment:
Don't we still need `secp256k1` here?

Reply:
Ah, yeah this is a good question. I removed this because the default nix build doesn't actually ship all of the headers. Maybe we can try and fix the nix build to include all of the modules. I can try adding it back in and reporting back with what errors end up firing. I have forgotten what they were, but IIRC they aren't fatal to the build process but they look very noisy and sus.